README.md

```diff
@@ -1,2 +1,37 @@
 # pms
-Pet Management System 
+
+Pet Management System
+
+## Project Idea
+
+Main idea reduce the number of abandoned pets in the world (Simple as this :) )
+
+Create a system that allow someone to search for pets that need adoption.
+This system should also provide an easy access to a Veterinarians and Pet Hospitals Network, making it simple to share data between them.
+This system should also allow pets to be advertised for adoption and where they are and their conditions. Mainly make things simple enough to facilitate the adoption of abandoned pets.
+Register former pets ouwner who abandoned a pet and place them in a blacklist. This will prevent pets from being placed/adopted by owners who won't really care about them.
+
+## Repository structure
+
+- backend-api -> will have the base api for the system to work
+- frontend -> will have the frontend application that will call the backend api
+- workder-services -> will have the background jobs that will manipulate, load and ingest data from both backend and frontend
+- infrastructure-services -> will have application or script that will help manage infrastructure (TBD: not sure if it worth)
```

Comment:
Maybe leave this out for now until we know what we want this to be?

Reply:
I agree I will mark it as something without priority or no need for now.